lib/managers/collaborations.js

```diff
@@ -119,7 +119,7 @@ Collaborations.prototype.respondToPending = function(collaborationID, newStatus,
  * @param {string} itemID - Box ID of the item to which the user should be invited
  * @param {CollaborationRole} role - The role which the invited collaborator should have
  * @param {Object} [options] - Optional parameters for the collaboration
- * @param {ItemType} [options.type=folder] - Type of object to be collaborated
+ * @param {ItemType} [options.type=folder] or [options.type=file]  - Type of object to be collaborated
```

Comment:
That's not how this JSDoc is used; the type `{ItemType}` gives all available values.  The [options.type=folder]` is just saying that the parameter is optional with a default value of `"folder"`.  Also, please add parameter descriptions for the new options you're adding!